src/Symfony/Bridge/Twig/Node/FormThemeNode.php

```diff
@@ -28,7 +28,7 @@ public function compile(Compiler $compiler)
     {
         $compiler
             ->addDebugInfo($this)
-            ->write('$this->env->getRuntime(\'Symfony\Bridge\Twig\Form\TwigRenderer\')->setTheme(')
+            ->write('$this->env->getRuntime(\'Symfony\Component\Form\FormRenderer\')->setTheme(')
```

Comment:
Possibly. Not sure if that needs to be handled. The only solution I see is to catch the `Twig_Error_Runtime` and then retry with the old TwigRenderer.